spec/controllers/api/v1/classifications_controller_spec.rb

```diff
@@ -154,6 +154,27 @@ def create_gold_standard
     end
   end
 
+  describe "#offset" do
+    let!(:first) { create(:classification, user: user, project: project) }
+    let!(:second) { create(:classification, user: user, project: project) }
+
+    before(:each) do
+      default_request user_id: authorized_user.id, scopes: scopes
```

Comment:
Can you check that a non-auth user cannot use this route too, ala [here](https://github.com/zooniverse/Panoptes/blob/50566215d7d263df52a307a2b433d2185623153f/app/controllers/api/v1/classifications_controller.rb#L5)